### PR TITLE
improvement(Employee registry)

### LINKED ISSUE #6263: Move the "Locked" column in Employee Registry to a line like the User Management interface
This is the way the employee registry looks by default:
image

We have a column for "locked" employees. Instead of having the locked column, let's do what User Management does and put a line through the employee if they are locked:

image

### PR DESCRIPTION
**Pull request by lomamech**
- Revome column locked
- Set text muted and add lock icone on employee display name

closes #6263

### CODE REVIEW

**Comment by lomamech** [comment]
image

File: client/src/modules/employees/registry/registry.js [hunk 1]
```diff
@@ -42,124 +46,136 @@ function EmployeeRegistryController(
...
       displayName          : 'TABLE.COLUMNS.REGISTRATION_NUMBER',
       headerCellFilter     : 'translate',
       footerCellClass      : 'text-center',
+      cellClass            : muteDisabledCells,
     },
     {
       field            : 'display_name',
       displayName      : 'TABLE.COLUMNS.NAME',
       headerCellFilter : 'translate',
+      cellTemplate     : '/modules/employees/templates/employee.cell.html',
```
Comment (jniles) [inline client/src/modules/employees/registry/registry.js:62]: You should add the same muted class to this cell if the employee is disabled.

**Review by jniles — COMMENTED** [review]
Looks pretty good!  Just one comment for consistency.

**Comment by lomamech** [comment]
@jniles i finish with this PR

File: client/src/modules/employees/registry/registry.js (hunk 1 above)
Comment (jniles) [reply client/src/modules/employees/registry/registry.js:62]: Can you make these grey too?

image

**Comment by lomamech** [comment]
image

**Comment by jniles** [comment]
LGTM.

Bors r+

**Pull request merged**